components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,387 @@
+          };
+          return (
+            <Form>
+              <Container flexDirection="column" justifyContent="center" px={[1, 30, 150]}>
+                <Container display="flex" flexDirection={['column', 'row', 'row']}>
+                  <Flex
+                    flexDirection="column"
+                    mx={[1, 10, 15]}
+                    width={[320, 350, 376, 476]}
+                    justifyContent="space-around"
+                  >
+                    <H4>
+                      <FormattedMessage id="organization.info.headline" defaultMessage="Organization's information" />
+                    </H4>
+                    <StyledInputField
+                      name="name"
+                      htmlFor="name"
+                      error={touched.name && errors.name}
+                      label={intl.formatMessage(messages.nameLabel)}
+                      value={values.name}
+                      onChange={handleSlugChange}
+                      required
+                      mt={4}
+                      mb={3}
+                      data-cy="cof-form-name"
+                    >
+                      {inputProps => (
+                        <Field as={StyledInput} {...inputProps} placeholder={intl.formatMessage(placeholders.name)} />
+                      )}
+                    </StyledInputField>
+                    <StyledInputField
+                      name="slug"
+                      htmlFor="slug"
+                      error={touched.slug && errors.slug}
+                      label={intl.formatMessage(messages.slugLabel)}
+                      value={values.slug}
+                      required
+                      mt={3}
+                      data-cy="cof-form-slug"
+                    >
+                      {inputProps => (
+                        <Field
+                          onChange={e => {
+                            setFieldValue('slug', e.target.value);
+                          }}
+                          as={StyledInputGroup}
+                          {...inputProps}
+                          prepend="opencollective.com/"
+                          placeholder={intl.formatMessage(placeholders.slug)}
+                        />
+                      )}
+                    </StyledInputField>
+                    {values.name.length > 0 && !touched.slug && (
+                      <P fontSize="10px">{intl.formatMessage(messages.suggestedLabel)}</P>
+                    )}
+                    <StyledInputField
+                      name="description"
+                      htmlFor="description"
+                      error={touched.description && errors.description}
+                      label={intl.formatMessage(messages.descriptionLabel)}
+                      required
+                      mt={3}
+                      data-cy="cof-org-description"
+                    >
+                      {inputProps => (
+                        <StyledTextarea
```

Comment:
Double check the placeholder and text alignment, text wrapping, and overflow of the `description` field.
<img width="1777" alt="Screenshot 2020-10-07 at 10 51 34" src="https://user-images.githubusercontent.com/37520401/95321217-a582eb00-0892-11eb-82e7-c20dc9d08bbb.png">

Reply:
Done